src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -157,21 +147,31 @@
     <RuntimeGroup Include="ol">
       <Parent>rhel</Parent>
       <Architectures>x64</Architectures>
-      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6</Versions>
+      <Versions>7;8</Versions>
+      <TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>
       <ApplyVersionsToParent>true</ApplyVersionsToParent>
     </RuntimeGroup>
-    <RuntimeGroup Include="ol">
-      <Parent>rhel</Parent>
+    <!-- The host doesn't strip minors for Oracle Linux.
+         We map them here to the major version or Oracle Linux and RHEL. -->
+    <RuntimeGroup Include="ol.7">
+      <Parent>rhel.7</Parent>
       <Architectures>x64</Architectures>
-      <Versions>8;8.0</Versions>
-      <ApplyVersionsToParent>true</ApplyVersionsToParent>
+      <Versions>0;1;2;3;4;6;6</Versions>
```

Comment:
We need to list out the minors because the host doesn't map them to a major, like it does for RHEL.
The previous definition required those minors to also be defined for RHEL.
This updated definition makes that unnecessary.